basins.py

```diff
@@ -203,36 +295,75 @@
 )
 
 
-
 basins = regionmask.Regions(
     [SO, ATL, PAC1, PAC2, ARC, IND, MED, BLK, HUD, BAL, RED, PER],
-    [ind_SO, ind_ATL, ind_tmp, ind_PAC, ind_ARC, ind_IND, ind_MED,
-     ind_BLK, ind_HUD, ind_BAL, ind_RED, ind_PER]
+    [
+        ind_SO,
+        ind_ATL,
+        ind_tmp,
+        ind_PAC,
+        ind_ARC,
+        ind_IND,
+        ind_MED,
+        ind_BLK,
+        ind_HUD,
+        ind_BAL,
+        ind_RED,
+        ind_PER,
+    ],
 )
 
 
+def generate_basin_codes(grid, lon="geolon", lat="geolat", mask="wet", persian=True):
```

Comment:
```suggestion
def generate_basin_codes(grid, lon="geolon", lat="geolat", mask="wet", persian=False):
```

We'll thank ourselves the next time we run CMIP if we set this to False now.